### PR TITLE
Add playlab mashup image

### PR DESCRIPTION
**Pull request by caleybrock**
Adding this image so we can update these pages:

screen shot 2015-12-01 at 10 02 48 pm
screen shot 2015-12-01 at 10 03 47 pm

Still need Alice/partner approval

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 1]
```diff
@@ -5,14 +5,14 @@ social:
   "twitter:card": photo
   "twitter:site": "@codeorg"
   "twitter:url": "http://code.org/playlab"
-  "twitter:image:src" : "https://code.org/images/playlab_original3.png"
+  "twitter:image:src" : "https://code.org/shared/images/courses/logo_tall_playlab.jpg"
```
Comment (Bjvanminnen) [inline pegasus/sites.v3/code.org/public/playlab.haml:8]: do we know whether /images and /shared have the same caching policy? maybe a question for @philbogle

**Comment by caleybrock** [comment]
chatted with phil/brent offline and it looks like these images are cached so we are good to go

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 2]
```diff
@@ -5,14 +5,14 @@ social:
...
   "twitter:title": "Play Lab Tutorials"
   "twitter:description": "Select between Gumball, Ice Age, and Classic themes to create your own story or game."
   "og:title": "Play Lab Tutorials"
   "og:description": "Select between Gumball, Ice Age, and Classic themes to create your own story or game."
-  "og:image" : "https://code.org/images/playlab_original3.png"
-  "og:image:width": "400"
-  "og:image:height": "400"
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/playlab.haml:15]: does twitter work okay with this dimension change?  any chance 400x400 was chosen specially?  i'd be inclined to use same dimensions if we're not sure.

Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/playlab.haml:15]: no these are for facebook and need to be exact

**Pull request merged**